examples/gtk+-3.0/Gtk.Builder.vala

```diff
@@ -0,0 +1,32 @@
+[GtkTemplate (ui = "/foo/bar/Gtk.Builder.ui")]
+public class BarWindow : Gtk.Window {
+	public string text {
+		get { return entry.text; }
+		set { entry.text = value; }
+	}
+
+	[GtkChild]
+	private Gtk.Entry entry;
+
+	[GtkCallback]
+	private void button_clicked (Gtk.Button button) {
+		message ("Button clicked, entry with text: %s", entry.text);
+	}
+
+	[GtkCallback]
+	private void entry_changed (Gtk.Editable editable) {
+		message ("Entry changed, text: %s\n", entry.text);
+
+		notify_property ("text");
+	}
+}
+
+public static int main (string[] args) {
```

Comment:
can just be `void main (string[] args)` and then get rid of the return statement below